web_src/css/markup/content.css

```diff
@@ -451,15 +444,16 @@
   text-decoration: inherit;
 }
 
-.markup pre > code {
-  padding: 0;
-  margin: 0;
-  font-size: 100%;
-  white-space: pre-wrap;
-  word-break: break-all;
-  overflow-wrap: break-word;
-  background: transparent;
-  border: 0;
+/* for inline code blocks */
+.markup p > code,
+.markup li > code,
+.markup p > tt,
+.markup li > tt {
+  padding: 0.2em 0.4em;
+  font-size: 85%;
+  white-space: break-spaces;
+  background-color: var(--color-markup-code-block);
+  border-radius: var(--border-radius);
 }
```

Comment:
I disagree with this selector list. what about `blockquote > code` etc, you can't possibly list every single element here.